providers/catalog.go

```diff
@@ -712,8 +712,8 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 				Upsert: false,
 				Delete: false,
 			},
-			Proxy:     false,
-			Read:      false,
+			Proxy:     true,
```

Comment:
can we revert `proxy` and `read` back to `false` until we've had a chance to do more thorough integration testing with the server?

Reply:
Removed. I thought that switching those on was a required task but something I had missed for read PR